python/lsst/ts/watcher/watcher_csc.py

```diff
@@ -95,12 +96,11 @@ async def close_tasks(self):
 
     async def configure(self, config):
         if self.model is not None:
-            # this should not happen, but in case it does shut down
-            # the old model (disable alarms first because that is synchronous
-            # and may avoid old alarms appearing due to a race condition)
+            # The model should be None, but if not, close it to get rid
+            # of the old alarms.
             self.log.warning("self.model unexpectedly present in configure method")
```

Comment:
Consider rephrasing this warning message a little bit to add more information. I'd suggest something like

```
"Model unexpectedly present in configuration stage. Closing and resetting it before continuing."
```